### PR TITLE
fix: revert "feat(feeshare): Developer Incentives (#144)"

### PR DESCRIPTION
**Pull request by inon-man**
This reverts commit 1e7936c622a6519596d302a8970b6f684ebd282b. This PR also includes orphan branch v2.1.1 adoption.

## Summary of changes
Fixes: #267 

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

File: x/feeshare/keeper/keeper.go
Comment (fragwuerdig) [inline x/feeshare/keeper/keeper.go:1]: Can we keep the `x/feeshare` folder for future use and just de-wire the module?

File: scripts/run-node.sh
[the remark this answers is not in the document]
Comment (fragwuerdig) [reply scripts/run-node.sh:1]: What you mean exactly? I am happy with renaming `HOME` to `HOME_DIR`

File: proto/terra/feeshare/v1beta1/feeshare.proto
Comment (fragwuerdig) [inline proto/terra/feeshare/v1beta1/feeshare.proto:1]: We should keep the `proto/terra/feeshare` folder for future use. no?

File: custom/wasm/keeper/handler_plugin.go
Comment (fragwuerdig) [inline custom/wasm/keeper/handler_plugin.go:1]: This is linting fix?

File: x/feeshare/keeper/testdata/reflect.wasm
Comment (fragwuerdig) [inline x/feeshare/keeper/testdata/reflect.wasm:1]: We need this occasionally for local tests. Please, do not remove.

**Review by fragwuerdig — CHANGES_REQUESTED** [review]
I'd rather keep `x/feeshare` directory.

File: x/feeshare/keeper/keeper.go
Comment (fragwuerdig) [reply x/feeshare/keeper/keeper.go:1]: Ok.

File: proto/terra/feeshare/v1beta1/feeshare.proto
Comment (fragwuerdig) [reply proto/terra/feeshare/v1beta1/feeshare.proto:1]: Resolved

File: x/feeshare/keeper/testdata/reflect.wasm
Comment (fragwuerdig) [reply x/feeshare/keeper/testdata/reflect.wasm:1]: Ok.

Comment (nghuyenthevinh2000) [reply x/feeshare/keeper/testdata/reflect.wasm:1]: I agree with inon-man, reflect.wasm can stay in contrib or script folder

**Review by nghuyenthevinh2000 — APPROVED** [review]

**Review by fragwuerdig — APPROVED** [review]

**Pull request merged**